ads/appnexus.js

```diff
@@ -108,14 +108,38 @@ function appnexusAst(global, data) {
     global.apntag = context.master.apntag;
   }
 
-  apntag.anq.push(() => {
-    apntag.onEvent('adAvailable', data.target, adObj => {
-      global.context.renderStart({width: adObj.width, height: adObj.height});
-      apntag.showTag(data.target, global.window);
+  // check for responses received before listeners are registered,
+  //  for example when an above-the-fold ad is scrolled into view
+  const tagsRequested = apntag.requests && apntag.requests.tags;
```

Comment:
eventually `apntag.requests.tags` will go away, is there a way to make this a more generic public API?